openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.activated.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:2635b9dd-0061-4059-ac61-2718ab366f75",
+  "type": "Event",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/ali",
+    "type": "Person"
+  },
+  "action": "Activated",
+  "object": {
+    "id": "course-v1:edx+cs-101+2018",
+    "type": "CourseOffering",
+    "username": "ali"
+  },
+  "eventTime": "2018-11-01T03:35:04.214775",
+  "referer": "http://localhost:18000/courses/course-v1:edx+cs-101+2018/about",
+  "session": "8205ce7ea40a0619955e15f072dc33e2",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
+      "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+      "event_source": "browser",
+      "event_type": "edx.course.enrollment.activated",
+      "host": "vagrant",
+      "ip": "10.0.2.2",
+      "name": "edx.course.enrollment.activated",
+      "page": null,
+      "event": {
```

Comment:
If we nest this data like in the current file then the data is duplicated. It is better to flatten these two objects and keep only one copy of the relevant data.

Reply:
fixed